### PR TITLE
Link to available plugins in Prow plugin config struct.

### PR DESCRIPTION
**Pull request by bweston92**

### CODE REVIEW

**Comment by bweston92** [comment]
/assign @fejta 

I hope this is what you meant by linking to available plugins.

**Comment by stevekuznetsov** [comment]
/ok-to-test
This is technically only true if that specific prow instance turns them all on

File: prow/plugins/config.go [hunk 1]
```diff
@@ -39,8 +39,7 @@ const (
 type Configuration struct {
 	// Plugins is a map of repositories (eg "k/k") to lists of
 	// plugin names.
-	// TODO: Link to the list of supported plugins.
-	// https://github.com/kubernetes/test-infra/issues/3476
+	// You can find comprehensive list of plugins here https://prow.k8s.io/command-help
```
Comment (fejta) [inline prow/plugins/config.go]: Wouldn't https://github.com/kubernetes/test-infra/tree/master/prow/plugins be a better link?

Also note that there can be external plugins as well

Comment (bweston92) [reply prow/plugins/config.go]: I agree it would provide more information, I got the link from #3476

**Pull request merged**